bokehjs/src/lib/models/text/utils.ts

```diff
@@ -5,14 +5,11 @@ import {BaseText} from "./base_text"
 export function is_tex_string(text: unknown): boolean {
   if (!isString(text)) return false
 
-  const dollars = "^\\$\\$.*?\\$\\$$"
-  const braces  = "^\\\[.*?\\\]$"
-  const parens  = "^\\\(.*?\\\)$"
+  const dollars = /^\\$\\$.*?\\$\\$$/m
+  const braces  = /^\\\[.*?\\\]$/m
+  const parens  = /^\\\(.*?\\\)$/m
```

Comment:
Multiline regex? I will make a quick guess that you want to trim/strip input text before matching (normally).